[Javascript][2.2.0] AMD define module paths fails in Webpack

<!--
Please follow the issue template below for bug reports and feature requests.
Also please indicate in the issue title which language/library is concerned. Eg:  [JAVA] Bug generating foo with bar 
-->
##### Description

Version 2.1.6 produced AMD paths with `./` prefixes. For example:

```
define(['./ApiClient', './model/Article', './model/ArticleResourceSingle', ... , './api/DefaultApi'], factory);
```

Version 2.2.0 produces AMD paths without `./` prefixes. For example: 

```
define(['ApiClient', 'model/Article', 'model/ArticleResourceSingle', ..., 'api/DefaultApi'], factory);
```

This has broken my webpack build.

```
ERROR in ../api/build/swagger/javascript/src/index.js
Module not found: Error: Cannot resolve module 'model/Article' in ../api/build/swagger/javascript/src
 @ ../api/build/swagger/javascript/src/index.js 28:4-701

ERROR in ../api/build/swagger/javascript/src/index.js
Module not found: Error: Cannot resolve module 'model/ArticleResourceSingle' in ../api/build/swagger/javascript/src
 @ ../api/build/swagger/javascript/src/index.js 28:4-701

```
##### Swagger-codegen version

<!-- which version of swagger-codegen are you using, is it a regression? -->

2.2.0
##### Swagger declaration file content or url

<!-- if it is a bug, a json or yaml that produces it.
    If you post the code inline, please wrap it with
    ```yaml
    (here your code)
    ```
    (for YAML code) or
    ```json
    (here your code)
    ```
    (for JSON code), so it becomes more readable. If it is longer than about ten lines,
    please create a Gist (https://gist.github.com) or upload it somewhere else and
    link it here.
  -->
##### Command line used for generation

<!-- including the language, libraries and various options -->
##### Steps to reproduce

<!-- unambiguous set of steps to reproduce the bug.-->
##### Related issues

<!-- has a similar issue been reported before? -->
##### Suggest a Fix

<!-- if you can't fix the bug yourself, perhaps you can point to what might be
  causing the problem (line of code or commit) -->

--- a/modules/swagger-codegen/src/main/resources/Javascript-es6/README.mustache
+++ b/modules/swagger-codegen/src/main/resources/Javascript-es6/README.mustache
@@ -32,7 +32,7 @@ npm install {{{projectName}}} --save
 ```
 
 #### git
-#
+
 If the library is hosted at a git repository, e.g.
 https://github.com/{{#gitUserId}}{{.}}{{/gitUserId}}{{^gitUserId}}YOUR_USERNAME{{/gitUserId}}/{{#gitRepoId}}{{.}}{{/gitRepoId}}{{^gitRepoId}}{{projectName}}{{/gitRepoId}}
 then install it via:
@@ -102,6 +102,31 @@ var callback = function(error, data, response) {
 api.{{{operationId}}}({{#allParams}}{{#required}}{{{paramName}}}{{#vendorExtensions.x-codegen-hasMoreRequired}}, {{/vendorExtensions.x-codegen-hasMoreRequired}}{{/required}}{{/allParams}}{{#hasOptionalParams}}{{#vendorExtensions.x-codegen-hasRequiredParams}}, {{/vendorExtensions.x-codegen-hasRequiredParams}}opts{{/hasOptionalParams}}{{#hasParams}}, {{/hasParams}}callback);
 {{/usePromises}}{{/-first}}{{/operation}}{{/operations}}{{/-first}}{{/apis}}{{/apiInfo}}
 ```
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error: Cannot resolve module", most certainly you should disable AMD loader:
+
+1. Add `imports-loader` package:
+
+    ```bash
+    npm install --save-dev imports-loader
+    ```
+2. Add the following sections to your webpack config (replace `MY_API_CLIENT_NAME` with your module name):
+
+    ```javascript
+    module: {
+      rules: [
+        {
+          test: /MY_API_CLIENT_NAME[\/\\]+.*\.js$/,
+          use: 'imports-loader?define=>false'
+        }
+      ]
+    },
+    node: {
+      fs: 'empty'
+    }
+    ```
 
 ## Documentation for API Endpoints
 

--- a/modules/swagger-codegen/src/main/resources/Javascript/README.mustache
+++ b/modules/swagger-codegen/src/main/resources/Javascript/README.mustache
@@ -32,7 +32,7 @@ npm install {{{projectName}}} --save
 ```
 
 #### git
-#
+
 If the library is hosted at a git repository, e.g.
 https://github.com/{{#gitUserId}}{{.}}{{/gitUserId}}{{^gitUserId}}YOUR_USERNAME{{/gitUserId}}/{{#gitRepoId}}{{.}}{{/gitRepoId}}{{^gitRepoId}}{{projectName}}{{/gitRepoId}}
 then install it via:
@@ -102,6 +102,31 @@ var callback = function(error, data, response) {
 api.{{{operationId}}}({{#allParams}}{{#required}}{{{paramName}}}{{#vendorExtensions.x-codegen-hasMoreRequired}}, {{/vendorExtensions.x-codegen-hasMoreRequired}}{{/required}}{{/allParams}}{{#hasOptionalParams}}{{#vendorExtensions.x-codegen-hasRequiredParams}}, {{/vendorExtensions.x-codegen-hasRequiredParams}}opts{{/hasOptionalParams}}{{#hasParams}}, {{/hasParams}}callback);
 {{/usePromises}}{{/-first}}{{/operation}}{{/operations}}{{/-first}}{{/apis}}{{/apiInfo}}
 ```
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error: Cannot resolve module", most certainly you should disable AMD loader:
+
+1. Add `imports-loader` package:
+
+    ```bash
+    npm install --save-dev imports-loader
+    ```
+2. Add the following sections to your webpack config (replace `MY_API_CLIENT_NAME` with your module name):
+
+    ```javascript
+    module: {
+      rules: [
+        {
+          test: /MY_API_CLIENT_NAME[\/\\]+.*\.js$/,
+          use: 'imports-loader?define=>false'
+        }
+      ]
+    },
+    node: {
+      fs: 'empty'
+    }
+    ```
 
 ## Documentation for API Endpoints
 

--- a/samples/client/petstore/javascript-es6/README.md
+++ b/samples/client/petstore/javascript-es6/README.md
@@ -24,7 +24,7 @@ npm install swagger_petstore --save
 ```
 
 #### git
-#
+
 If the library is hosted at a git repository, e.g.
 https://github.com/GIT_USER_ID/GIT_REPO_ID
 then install it via:
@@ -68,6 +68,31 @@ var callback = function(error, data, response) {
 api.fakeOuterBooleanSerialize(opts, callback);
 
 ```
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error: Cannot resolve module", most certainly you should disable AMD loader:
+
+1. Add `imports-loader` package:
+
+    ```bash
+    npm install --save-dev imports-loader
+    ```
+2. Add the following sections to your webpack config (replace `MY_API_CLIENT_NAME` with your module name):
+
+    ```javascript
+    module: {
+      rules: [
+        {
+          test: /MY_API_CLIENT_NAME[\/\\]+.*\.js$/,
+          use: 'imports-loader?define=>false'
+        }
+      ]
+    },
+    node: {
+      fs: 'empty'
+    }
+    ```
 
 ## Documentation for API Endpoints
 

--- a/samples/client/petstore/javascript-promise-es6/README.md
+++ b/samples/client/petstore/javascript-promise-es6/README.md
@@ -24,7 +24,7 @@ npm install swagger_petstore --save
 ```
 
 #### git
-#
+
 If the library is hosted at a git repository, e.g.
 https://github.com/GIT_USER_ID/GIT_REPO_ID
 then install it via:
@@ -65,6 +65,31 @@ api.fakeOuterBooleanSerialize(opts).then(function(data) {
 
 
 ```
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error: Cannot resolve module", most certainly you should disable AMD loader:
+
+1. Add `imports-loader` package:
+
+    ```bash
+    npm install --save-dev imports-loader
+    ```
+2. Add the following sections to your webpack config (replace `MY_API_CLIENT_NAME` with your module name):
+
+    ```javascript
+    module: {
+      rules: [
+        {
+          test: /MY_API_CLIENT_NAME[\/\\]+.*\.js$/,
+          use: 'imports-loader?define=>false'
+        }
+      ]
+    },
+    node: {
+      fs: 'empty'
+    }
+    ```
 
 ## Documentation for API Endpoints
 

--- a/samples/client/petstore/javascript-promise/README.md
+++ b/samples/client/petstore/javascript-promise/README.md
@@ -24,7 +24,7 @@ npm install swagger_petstore --save
 ```
 
 #### git
-#
+
 If the library is hosted at a git repository, e.g.
 https://github.com/GIT_USER_ID/GIT_REPO_ID
 then install it via:
@@ -65,6 +65,31 @@ api.fakeOuterBooleanSerialize(opts).then(function(data) {
 
 
 ```
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error: Cannot resolve module", most certainly you should disable AMD loader:
+
+1. Add `imports-loader` package:
+
+    ```bash
+    npm install --save-dev imports-loader
+    ```
+2. Add the following sections to your webpack config (replace `MY_API_CLIENT_NAME` with your module name):
+
+    ```javascript
+    module: {
+      rules: [
+        {
+          test: /MY_API_CLIENT_NAME[\/\\]+.*\.js$/,
+          use: 'imports-loader?define=>false'
+        }
+      ]
+    },
+    node: {
+      fs: 'empty'
+    }
+    ```
 
 ## Documentation for API Endpoints
 

--- a/samples/client/petstore/javascript/README.md
+++ b/samples/client/petstore/javascript/README.md
@@ -24,7 +24,7 @@ npm install swagger_petstore --save
 ```
 
 #### git
-#
+
 If the library is hosted at a git repository, e.g.
 https://github.com/GIT_USER_ID/GIT_REPO_ID
 then install it via:
@@ -68,6 +68,31 @@ var callback = function(error, data, response) {
 api.fakeOuterBooleanSerialize(opts, callback);
 
 ```
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error: Cannot resolve module", most certainly you should disable AMD loader:
+
+1. Add `imports-loader` package:
+
+    ```bash
+    npm install --save-dev imports-loader
+    ```
+2. Add the following sections to your webpack config (replace `MY_API_CLIENT_NAME` with your module name):
+
+    ```javascript
+    module: {
+      rules: [
+        {
+          test: /MY_API_CLIENT_NAME[\/\\]+.*\.js$/,
+          use: 'imports-loader?define=>false'
+        }
+      ]
+    },
+    node: {
+      fs: 'empty'
+    }
+    ```
 
 ## Documentation for API Endpoints
 

